### ISSUE TITLE
Replace `vue-sweetalert2` with `vue-material` dialogs

### ISSUE THREAD

**Issue by dmohns** [EnAccess/micropowermanager#174]
We are using SweetAlert to display dialogs which just adds complexity and dependencies.

Please change to use a `md-dialog` instead: https://www.creative-tim.com/vuematerial/components/dialog

**Comment by munyanezaarmel** [comment]
changing in the codebase where we used this.$swal to md-dialog?

**Comment by dmohns** [comment]
Yeah, so the definition of done here is that we can remove `vue-sweetalert2` from the dependencies using `npm uninstall vue-sweetalert2`.

However, the steps to achieve this aren't quite clear (to me) and have to be developed as part of this ticket.
